### ISSUE TITLE
How to define small-scale

### ISSUE THREAD

**Issue by ocmykr2** [yaoxingcheng/TLM#15]
Hello! Can I ask what's the difference between small-scale and mideum-scale. Is it different because of  the different values of k?
And I also get f1 = 0.7687 instead of 0.8051 in small-scale Sci-ERC. Why would that happen?
Btw,this is a great framework! Thank you!

**Comment by yaoxingcheng** [comment]
Hi~ Thanks for your interest in our work. 
The difference between scales in TLM is essentially the hyper-parameters we choose that affect the training FLOPs (such as batch size, training steps, sequence length). The detailed hyper-parameters for each scale can be found in Table B.2 in our paper v1.
For tasks with low resources such as SciERC, difference in hardware and random seeds may lead to seemingly different evaluation results. But 0.7687 does seem exceptionally low for SciERC. My suggestion is 1) make sure the hyperparameters you use are aligned with that mentioned in the paper (exactly in those provided example scripts) 2) proper model selection: during finetuning we will evaluate the model several times, make sure the test results are from the one with the best dev set result.

**Comment by ocmykr2** [comment]
WoW!
Now I have understood the scales' meaning. (I read the table but somehow misunderstand it ^_^.
And For the second question, I found myself have mistaken **test** marco_f1 to be the result.
Thanks for your detailed answers!

**Comment by yaoxingcheng** [comment]
You are welcome. But we do use **test marco_f1** as the reported metric in the main results. The point is we don't use test set results evaluated in the final step, but use the test set results when the best dev set result is achieved. 
After training is complete, there should be a file called `trainer_state.json` in the output directory, and there's a field called `test_results` in that file, the number there ought to be accurate. You may check it out to see whether there's an issue.
